CI: ConformanceKind1.19: installation-and-connectivity fails on "Enable Relay" for second test

Example run: https://github.com/cilium/cilium/runs/6679723958?check_suite_focus=true#logs

While running the workflow, the first test without encryption passes, then Cilium apparently cleans up. However, the `hubble-peer` service is not removed. Then when installing Cilium a second time, this time with encryption, enabling relay fails with:

```
🚀 Creating Peer Service...

Error: Unable to enable Hubble:  services "hubble-peer" already exists
Error: Process completed with exit code 1.
```

--- a/install/kubernetes/cilium/templates/validate.yaml
+++ b/install/kubernetes/cilium/templates/validate.yaml
@@ -35,7 +35,9 @@
 {{- end }}
 
 {{/* validate hubble-ui specific config */}}
-{{- if .Values.hubble.ui.enabled }}
+{{- if and .Values.hubble.ui.enabled
+  (ne .Values.hubble.ui.backend.image.tag "latest")
+  (ne .Values.hubble.ui.frontend.image.tag "latest") }}
   {{- if regexReplaceAll "@.*$" .Values.hubble.ui.backend.image.tag "" | trimPrefix "v" | semverCompare "<0.9.0" }}
     {{ fail "Hubble UI requires hubble.ui.backend.image.tag to be '>=v0.9.0'" }}
   {{- end }}

